sw/device/silicon_creator/rom/rom.c

```diff
@@ -155,7 +154,7 @@ static rom_error_t rom_init(void) {
   SEC_MMIO_WRITE_INCREMENT(kFlashCtrlSecMmioInit);
 
   // Initialize in-memory copy of the ePMP register configuration.
-  rom_epmp_state_init(&epmp, lc_state);
+  rom_epmp_state_init((epmp_state_t *)&epmp_state, lc_state);
```

Comment:
This is what I'm hoping to avoid by reaching a consensus on some guidelines for using `volatile`. It's contagious and we are not supposed to cast it away here if we want to be consistent. If something is `volatile` we cannot use `mem*` functions either. We can leave this as-is for now since we will work on it with @luismarques.

Reply:
Yeah, I feel the same.